blackbox/docs/cli-tools.rst

```diff
@@ -101,5 +108,111 @@ The CrateDB process can handle the following signals.
     the node is stopped, you can perform a `graceful stop`_ with the
     :ref:`DECOMMISSION <alter_cluster_decommission>` statement instead.
 
+Crate-node
+==========
+
+Managing clusters
+-----------------
+
+This section shows you how to manage CrateDB clusters from the command line.
+
+.. SEEALSO::
+
+   For step-by-step how-tos and examples on how to troubleshoot CrateDB clusters and
+   nodes from the command line, please refer to `Troubleshooting with crate-node CLI`_.
+
+Introduction
+------------
+
+CrateDB provides a ``crate-node`` command, in the ``bin`` directory, that lets
+you form new clusters, repair broken clusters, and repair broken nodes.
+
+With this command you can:
+
+ * Unsafely bootstrap a cluster
+ * Detach a node from its cluster
+ * Change the role of a node
+
+.. NOTE::
```

Comment:
move this to the commands section

also please make sure to indent admonition blocks by 4 spaces